### PR TITLE
Clarify ingress and egress IP address in allowlist

### LINKED ISSUE #901: permissioning smart contracts - extra info re network config / multiple IPs
Add some extra info that you might need to add "ingress and egress" IPs to permissioning allowlists and also if your network config changes eg with k8s https://github.com/ConsenSys/permissioning-smart-contracts/issues/257

Check the issue type in the following list (insert X instead of space between [ ]): -->

- [x] Missing content
- [ ] Outdated content
- [ ] Wrong content
- [ ] Confusing or misleading content
- [ ] Other

### PR DESCRIPTION
**Pull request by rolandtyler**
Signed-off-by: Roland Tyler <roland.tyler@consensys.net>

## Describe the change
Add clarification when using a node with ingress and egress IP address for allowlist. 
Add clarification about updating node allowlists after changing network config.

## Issue fixed
Fixes #901 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

## Testing
https://hyperledger-besu--919.org.readthedocs.build/en/919/HowTo/Limit-Access/Updating-Permission-Lists/

https://hyperledger-besu--919.org.readthedocs.build/en/919/HowTo/Limit-Access/Local-Permissioning/

https://hyperledger-besu--919.org.readthedocs.build/en/919/Concepts/Permissioning/Onchain-Permissioning/

### CODE REVIEW

**Review by NicolasMassart — APPROVED** [review]
Looks good to me.

File: docs/HowTo/Limit-Access/Local-Permissioning.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ start with node permissions enabled.
     The `nodes-allowlist` in the [permissions configuration file](#permissions-configuration-file)
     must contain the specified bootnodes.
 
+!!! tip
+
+    If your node has two different IP addresses for ingress and egress
+    (for example, if you use Kuberetes implementing a load balancer for ingress and a NAT gateway IP address for egress),
```
Comment (macfarla) [inline docs/HowTo/Limit-Access/Local-Permissioning.md:54]: s/Kuberetes/Kubernetes

File: docs/HowTo/Limit-Access/Updating-Permission-Lists.md [hunk 2]
```diff
@@ -14,7 +14,13 @@ To add a node to the Hyperledger Besu nodes allowlist:
 1. On the **Nodes** tab of the [permissioning management dapp](../../Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md),
    select **Add Node**.
    The **Add Node** window displays.
-1. Enter the [enode URL](../../Concepts/Node-Keys.md#enode-url) of the node you are adding and select **Add Node**.
+2. Enter the [enode URL](../../Concepts/Node-Keys.md#enode-url) of the node you are adding and select **Add Node**.
+
+!!! tip
+
+    If your node has two different IP addresses for ingress and egress
+    (for example, if you use Kuberetes implementing a load balancer for ingress and a NAT gateway IP address for egress),
```
Comment (macfarla) [inline docs/HowTo/Limit-Access/Updating-Permission-Lists.md:22]: s/Kuberetes/Kubernetes

Comment (macfarla) [reply docs/HowTo/Limit-Access/Updating-Permission-Lists.md:22]: can we add Kubernetes to the dictionary?

**Review by macfarla — APPROVED** [review]
approved once typo is fixed

**Pull request merged**